src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -87,24 +90,30 @@ const withGraphQLMutation = (...args: Parameters<typeof graphql.mutation>) => ({
 
 const withBuilds = ({
   lastBuildOnBranch,
-  selectedBuild,
+  selectedBuild: getSelectedBuild,
   userCanReview = true,
 }: {
-  selectedBuild?: SelectedBuildFieldsFragment;
+  selectedBuild?:
+    | SelectedBuildFieldsFragment
+    | ((selectedBuildId: string | undefined) => SelectedBuildFieldsFragment);
   lastBuildOnBranch?: LastBuildOnBranchBuildFieldsFragment;
   userCanReview?: boolean;
 }) => {
-  return withGraphQLQueryResult(QueryBuild, {
-    project: {
-      name: "acme",
-      lastBuildOnBranch: lastBuildOnBranch || selectedBuild,
-    },
-    selectedBuild,
-    viewer: {
-      projectMembership: {
-        userCanReview,
+  return withGraphQLQueryResult(QueryBuild, ({ selectedBuildId }) => {
+    const selectedBuild =
+      typeof getSelectedBuild === "function" ? getSelectedBuild(selectedBuildId) : getSelectedBuild;
```

Comment:
I agree if there's a clear way to update all the stories to use the same 

Instead of every story passing in a function, since we know the behavior is to get the last build on branch by id, maybe we could just write the same function you passed in the new story here.
This way we only add the method once. The problem is we need to pass an ID on the lastBuildOnBranch in a story for it to work how we want.
```suggestion
      selectedBuildId === lastBuildOnBrach.id ? lastBuildOnBranch : selectedBuild
```